src/utils/numbers.ts

```diff
@@ -1,13 +1,19 @@
 import {Language} from '@atb/translations';
 
-/*
-Utility method for formatting a decimal number. Will use comma/dot based on
-language, and a maximal of two decimals. Will not round the decimals. The
-fraction digits count will be fixed if parameter fractionDigits is provided.
+export const formatFarePrice = (
+  str: number,
+  language: Language,
+  fractionDigits?: number,
+  isVat?: boolean /**Always show 2 decimals in VAT value*/,
+) => {
+  const minFractionDigits = isVat ? 2 : str % 1 == 0 ? 0 : 2;
+  return str.toLocaleString(language, {
+    minimumFractionDigits: minFractionDigits,
+    maximumFractionDigits: fractionDigits || 2,
+    useGrouping: false,
+  });
+};
```

Comment:
The implementation looks good, but the naming could be improved. It is slightly nitpicking, but I think it is good learning in the thought behind it :)

As this is a generic utility function it shouldn't use domain specific terms like 'fare' or 'vat'. Instead it should be named according to what it does. So the field `isVat` should be instead `alwaysShowTwoDecimals` or something like it. Also in the name of the function the 'fare' part should be changed to a wording which reflects what is special about the price formatting for fares as opposed to other prices.